index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>SergSkol Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <header class="container header_flex">
+        <div class="logo">My Logo</div>
+        <div class="logo">≡</div>
+    </header>
+    <section class="shapes">
+        <img class="rectangle4" src="./img/Rectangle 4.png" alt="">
+        <img class="elipse3" src="./img/Ellipse 3.png" alt="">
```

Comment:
- [ ] The Figma tempate you chose has a decoration background image which should be implemented in your project with the use of css background-image property inside your css file not with the use of `img` tag. Kindly check the figma design and fix that area.

Reply:
Thank you! Done!